enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -8160,18 +8239,25 @@ class AdjointGenerator
           getForwardBuilder(Builder2);
 
           SmallVector<Value *, 2> args;
-#if LLVM_VERSION_MAJOR >= 14
-          for (unsigned i = 0; i < orig->arg_size(); ++i)
-#else
-          for (unsigned i = 0; i < orig->getNumArgOperands(); ++i)
-#endif
-          {
+
+          for (unsigned i = 0; i < orig->getNumArgOperands(); ++i) {
```

Comment:
Please restore the LLVM14 check (it hasn't been released yet / packed for Ubuntu in CI, but let's not break it until it does if we can).

Reply:
merge error ...